apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingFactory.kt

```diff
@@ -169,6 +169,12 @@ class BehandlingFactory(
                 ) ?: throw IllegalStateException("Kunne ikke opprette behandling")
             }.behandling
 
+        if (request.kilde == Vedtaksloesning.GJENOPPRETTA) {
+            oppgaveService.hentOppgaverForSak(sak.id).find { it.referanse == "TOM" }?.let {
```

Comment:
Denne kjens litt hacky, har vi ein annan måte å skilje på her?

Reply:
Jeg klarte ikke å finne en måte som ikke føltes kjip ut egentlig. Jeg vurderte å legge til en ny oppgavetype eller lignende men litt dumt å lage en type som er så midlertidig.

Reply:
Nei, ikkje sant. Ser den. Foreslår ein konstant for tekstreferansen, det hjelper litt iallfall på magekjensla.

Reply:
Ikke dumt. Var allerede mismatch på strengene 😓 Tom/mangler.

Reply:
Først så tenkte jeg bare tom streng men var redd for at det kunne oppstå i andre sammenhenger.

Reply:
Ja, det ser eg også for meg kan skje, så det er fint med noko spesifikt.